modules/data/symbols/patches/language/pokeleafgreen.yml

```diff
@@ -0,0 +1,645 @@
+---
+#--------------------------------#
+#    Overworld / Util Symbols    #
+#--------------------------------#
+CB2_OVERWORLD:
+  D: 0x80565d4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_YesNoMenu_HandleInput:
+  D: 0x809cf0c
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_FlyIntoMap:
+  D: 0x8084330
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+Task_ExitDoor:
+  D: 0x807def4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+WaitForAorBPress:
+  D: 0x806b858
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+EventScript_UseStrength:
+  D: 0x81c1bc4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+#-------------------------------#
+
+#--------------------#
+#    Map Symbols     #
+#--------------------#
+gMapGroups:
+  D: 0x83525ac
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+sMapNames:
+  D: 0x83f1340
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gWildMonHeaders:
+  D: 0x83c9940
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gSaveBlock1Ptr:
+  D: 0x3004f58
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gSaveBlock2Ptr:
+  D: 0x3004f5c
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_RegionMap:
+  D: 0x80c0a58
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gMapHeader:
+  J: ~
+gObjectEvents:
+  J: ~
+gPlayerAvatar:
+  J: ~
+gPlayerPartyCount:
+  J: ~
+gPlayerParty:
+  J: ~
+sMapCursor:
+  J: ~
+#----------------#
+
+#--------------------#
+#  Battling Symbols  #
+#--------------------#
+CB2_OVERWORLDBASIC:
+  D: 0x80565c8
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_INITBATTLE:
+  D: 0x800fd20
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_HANDLESTARTBATTLE:
+  D: 0x801048c
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_RETURNTOFIELD:
+  D: 0x80567fc
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+CB2_RETURNTOFIELDCONTINUESCRIPTPLAYMAPMUSIC:
+  D: 0x80568e4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gMain:
+  D: 0x3003040
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gBattleMainFunc:
+  D: 0x3004ed4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+gBattlerControllerFuncs:
+  D: 0x3004f30
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+GTASKS:
+  D: 0x3004fe0
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+BattleMainCB1:
+  D: 0x8012368
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+BattleMainCB2:
+  D: 0x8011084
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+HandleTurnActionSelectionState:
+  D: 0x8013fc4
+  F: ~
+  I: ~
+  J: ~
+  S: ~
+HandleInputChooseAction:
+  D:
+    - ~
+    - ~
+    - 0x802e3d0
+    - ~
+  F:
+    - ~
+    - ~
+    - ~
+    - ~
+  I:
+    - ~
+    - ~
+  J: ~
+  S:
+    - ~
+    - ~
+    - ~
+    - ~
```

Comment:
Empty lists are kinda confusing, I think it should be better to use empty value instead